content/v2.0/reference/api/influxdb-1x/dbrp.md

```diff
@@ -0,0 +1,155 @@
+
+The DBRP mapping service uses the **database** and **retention policy** specified in
+[compatibility API](/v2.0/reference/api/influxdb-1x/) requests to route operations to a bucket.
+[Create DBRP mappings automatically](#automatically-create-dbrp-mappings) with the `/write` compatibility endpoint or
+[create DBRP mappings manually](#manually-create-and-manage-dbrp-mappings) using the `/api/v2/dbrps` API endpoint.
+
+- [DBRP mapping behavior](#dbrp-mapping-behavior)
+- [Automatically create DBRP mappings](#automatically-create-dbrp-mappings)
+- [Manually create and manage DBRP mappings](#manually-create-and-manage-dbrp-mappings)
+
+### Default retention policies
+A database can have multiple retention policies with one set as default.
+If no retention policy is specified in a query or write request, InfluxDB uses
+the default retention policy for the specified database.
+
+## DBRP mapping behavior
+
+- [When writing data](#when-writing-data)
+- [When querying data](#when-querying-data)
+
+### When writing data
+When writing data to InfluxDB 2.0 using the [`/write` compatibility endpoint](/v2.0/reference/api/influxdb-1x/write/),
+the DBRP mapping service uses the specified database and retention policy to:
+
+1. Check for a DBRP mapping that matches the database and retention policy.
+  - If a DBRP mapping exists, write data to the mapped bucket.
+  - If no mapping exists, then:
+2. Check to see if a bucket matching the naming convention, `<database>/<retention-policy>`, exists.
+  - If a bucket with that name exists, add the DBRP mapping to the existing bucket
+     and write the data to the bucket.
+  - If no bucket with that name exists, then:
+3. Create a new bucket using the naming convention `<database>/<retention-policy>`,
+   add the DBRP mapping to the new bucket, and write the data to the bucket.
+
+##### If no retention policy is specified:
+1. Check for the specified database in the DBRP mapping service.
+  - If the database exists, get the default retention policy for the specified database, and then:
+  - If the database doesn't exists, add the database, set its default retention policy as `autogen`, and then:
+2. Continue the [normal DBRP mapping behavior](#when-writing-data).
+
+### When querying data
+When querying data from InfluxDB 2.0 using the [`/query` compatibility endpoint](/v2.0/reference/api/influxdb-1x/query/),
+the DBRP mapping service uses the specified database and retention policy to:
+
+Check for a DBRP mapping that matches the database and retention policy.
+
+- If a DBRP mapping exists, query data from the mapped bucket.
+- If no mapping exists, return an error.
+
+##### If no retention policy is specified:
+1. Check for the specified database in the DBRP mapping service.
+  - If the database doesn't exists, return an error.
+  - If database exists, get the default retention policy for the specified database, and then:
+2. Continue the [normal DBRP mapping behavior](#when-querying-data).
```

Comment:
the DBRP mapping service checks for the specified database and retention policy (if no retention policy is specified, the default database retention policy is used):

- If a mapped bucket exists, data is queried from the mapped bucket.
- If no mapped bucket exists, an error is returned.